pubtools/_pulp/tasks/publish.py

```diff
@@ -0,0 +1,190 @@
+import logging
+import sys
+import re
+from datetime import datetime
+from functools import partial
+from more_executors.futures import f_map, f_sequence
+
+from pubtools.pulplib import Criteria, Matcher, PublishOptions
+
+from pubtools._pulp.task import PulpTask, CDNCached
+from pubtools._pulp.services import PulpClientService, UdCacheClientService
+
+
+step = PulpTask.step
+
+LOG = logging.getLogger("pubtools.pulp")
+
+
+def publish_date(str_date):
+    # validates publish-before date
+    return datetime.strptime(str_date, "%Y-%m-%d")
+
+
+class Publish(PulpClientService, UdCacheClientService, PulpTask, CDNCached):
+    """Publishes the pulp repositories to the endpoints defined by the distributors
```

Comment:
Let's try to keep the writing style consistent in the docs.

clear-repo https://release-engineering.github.io/pubtools-pulp/clear-repo.html for instance is:

> Remove all contents from one or more Pulp repositories.

Therefore shouldn't this be:

```suggestion
    """Publish one or more Pulp repositories to the endpoints defined by their distributors.
```

In particular the differences are:

- Written in the imperative mood ("Publish ..." and not "Publishes ...")
- Sentences end with a period `.`
- `pulp` is written as `Pulp` (capitalize proper nouns)

Ideally, the docs for different commands are not going to look different based on whichever dev happened to implement the command.

Reply:
+1